### DISCUSSION TITLE
Trying to print a temp tower, print starts cold?

### DISCUSSION THREAD

**Discussion by QuartzIMG** [supermerill/SuperSlicer#1844] [Q&A] [answered]
When i try to print this temp tower, the print starts with the printer dead cold with no warmup.

I'm starting from Calibration > Temp, and setting it up like this:
![image](https://user-images.githubusercontent.com/27329/141027076-9d374dec-a7fc-4f50-a473-24cd9e2fe537.png)

But in two different gcode senders it starts to print cold, have I borked some filament setting somewhere? first and subsequent layers should be 210 with what i have set:

![image](https://user-images.githubusercontent.com/27329/141027205-3df4c03f-c46c-4b26-8289-4823705a1cdf.png)

Attached the resulting gcode, when i view it in temp mode it LOOKS like its trying to print that first layer cold: 
![image](https://user-images.githubusercontent.com/27329/141027412-80b7036e-f3d8-4796-a0ed-90566b505362.png)
 
 what have i done?
[Temperature calibration.gcode.txt](https://github.com/supermerill/SuperSlicer/files/7508927/Temperature.calibration.gcode.txt)

**Comment by QuartzIMG** [comment]
Ok this is weirder to me, only ONE of the flow calibration parts tries to print cold? O.o 
![image](https://user-images.githubusercontent.com/27329/141027635-1ed59fc7-9c4e-48cd-8d2e-18f3b6f5bb10.png)

**Reply by supermerill** [reply]
only one because they are printed one after another (complete_object setting) So only the first one start cold.

**Comment by supermerill** [comment]
The only way to start cold is to set the "only custom start gcode" and mess up the custom start gcode (as i've added a failsafe to prevent any cold start).

**Comment by QuartzIMG** [comment, marked as answer]
@supermerill Hey thanks for the fast reply! I dont think i changed that at all: 
![image](https://user-images.githubusercontent.com/27329/141028933-98768d8e-7ac3-47cd-9016-90cd157858bd.png)


attached the gcode as well, i suck at reading it but i see the initial setting of the bed and extruder temps  
![image](https://user-images.githubusercontent.com/27329/141029047-304fb8a0-eb76-493a-aa2b-3fd07acea072.png)
[Temperature calibration.gcode.txt](https://github.com/supermerill/SuperSlicer/files/7508939/Temperature.calibration.gcode.txt)

**Reply by supermerill** [reply]
show me your filament custom gcode

**Reply by QuartzIMG** [reply]
![image](https://user-images.githubusercontent.com/27329/141029574-8ecf2bf8-45f2-4df7-ad81-7c0f7110a97a.png)

**Reply by supermerill** [reply]
the problem in yoru gcode is the "M109 S65 ;wauit for bed temp...."
as M109 is for extruder.
But where does it comes?

**Reply by supermerill** [reply]
oh, maybe you set your firmware to be makerware or sailfish?

**Reply by QuartzIMG** [reply]
This is my entire user config, if thats useful? i just have the one profile, only started testing with superslicer heavily today (its awesome by the way
[SuperSlicer.zip](https://github.com/supermerill/SuperSlicer/files/7508981/SuperSlicer.zip)
)

**Reply by QuartzIMG** [reply]
Ooops! 
![image](https://user-images.githubusercontent.com/27329/141029998-e1bd3270-3d84-4ddc-9919-b3b87a70bcdd.png)
 Ill try Marlin!

**Reply by QuartzIMG** [reply]
That was it, sorry to trouble you about it! Drove me NUTS 
![image](https://user-images.githubusercontent.com/27329/141030119-01ab24a4-901f-4d21-af9e-07521c2601fb.png)

**Reply by supermerill** [reply]
it's that. your makerbot use M109 to heat bed, not extruder, but the current gcode viewer isn't capable of understanding that. that's why you see that.
It's a kind of bug (or oversight)
